hotspot/src/os/linux/vm/osContainer_linux.hpp

```diff
@@ -66,8 +65,7 @@ class OSContainer: AllStatic {
 };
 
 inline bool OSContainer::is_containerized() {
-  assert(_is_initialized, "OSContainer not initialized");
```

Comment:
This hunk got removed with `JDK-8229202` which is not yet part of 8u. Please add this issue by:

```
/issue add JDK-8229202
```